### ISSUE TITLE
Bundle report

### ISSUE THREAD

**Issue by firelizzard18** [evanw/esbuild#1568]
I would like to be able to generate a report from esbuild showing what files were bundled and how much each file contributed to the final bundled size. For example, if I have some `main.js` file that imports `is-buffer`, I'd like to see something like:

```
  out/main.js                        123b
  ├ src/main.js                      12b
  └ node_modules/is-buffer/index.js  23b
```

**Comment by hyrious** [comment]
[metafile](https://esbuild.github.io/api/#metafile)

**Comment by evanw** [comment]
Yes the metafile should have all of the information you need. You can use it to write some code that does what you want.

**Comment by firelizzard18** [comment]
For anyone else trying to do this, the following (linux/mac) shell commands will output a list of "<bytes in output> <from file>":

```shell
cat meta.json | jq -r '.outputs."out/extension.js".inputs | to_entries[] | "\(.value.bytesInOutput)\t\(.key)"' | sort -k 1 -nr
```
